app/api/__init__.py

```diff
@@ -100,9 +101,13 @@
 api.route(EmailNotificationRelationshipOptional, 'email_notification_event',
           '/email-notifications/<int:id>/relationships/event')
 
-# image_sizes
-api.route(ImageSizeList, 'image_size_list', '/image-sizes')
-api.route(ImageSizeDetail, 'image_size_detail', '/image-sizes/<int:id>')
+# event_image_sizes
+api.route(EventImageSizeDetail, 'event_image_size_detail', '/event-image-sizes/<id>',
+          '/event-image-sizes')
```

Comment:
`'/event-image-sizes/<id>'` why do we need an `id` in the route when we know that will always be 1 ?

Reply:
Refer to the Modules API